pages/games.vue

```diff
@@ -29,7 +70,27 @@ export default {
   },
   components: {
     GamesTable,
-    AntHivePageHeader
+    AntHivePageHeader,
+    AntHiveIcon
+  },
+  data() {
+    return {
+      gamesSortPreset: 'best-score',
```

Comment:
why you use 'preset' ? is this a common word for sorting type ?
maybe gamesSortType is better.
what you think ?